sympy/polys/matrices/ddm.py

```diff
@@ -483,5 +486,68 @@ def is_lower(self):
         zero = self.domain.zero
         return all(Mij == zero for i, Mi in enumerate(self) for Mij in Mi[i+1:])
 
+    def lll(A, delta: Rational = Rational(3, 4)) -> 'DDM':
```

Comment:
We should avoid the use of Rational here. Again this is a problem of different levels. We are working at the domain level so this could be `QQ` or it could also be a float. Here 0.75 is an exact float or more generally if we want exactness we can ask the user to supply `a` and `b` so that `delta = a/b` or something.

From what I can tell the exact value of delta is not so important so maybe we can just take a float like python_flint does.